docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/ConvertToImage.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class ConvertToImage
+    {
+        private const int inputSize = 3 * 224 * 224;
+
+        private class DataPoint
+        {
+            [VectorType(inputSize)]
+            public double[] Features { get; set; }
+        }
+
+        private sealed class TransformedData
+        {
+            public float[] Pixels { get; set; }
+        }
+
+        private static IEnumerable<DataPoint> GenerateRandomDataPoints(int count, int seed = 0)
+        {
+            var random = new Random(seed);
+
+            for (int i = 0; i < count; i++)
+            {
+                yield return new DataPoint
+                {
+                    Features = Enumerable.Repeat(0, inputSize).Select(x => random.NextDouble() * 100).ToArray()
+                };
+            }
+        }
```

Comment:
i'd move this below the example. If the users want to scroll to it they can.

Reply:
fixed

---
In reply to: [271135840](https://github.com/dotnet/machinelearning/pull/3165#discussion_r271135840) [](ancestors = 271135840)